server/src/diagnostic_ext.rs

```diff
@@ -52,11 +45,17 @@ pub async fn compile_and_publish_diagnostics(
 ) {
     let mut configuration_sets = configuration_sets.lock().await;
 
+    client
+        .log_message(
+            MessageType::INFO,
+            "Publishing diagnostics for all configuration sets..",
```

Comment:
An ellipsis usually has 3 periods in it.
```suggestion
            "Publishing diagnostics for all configuration sets...",
```